uint8 type
uint8 Custom=254
uint8 CDouble=1
uint8 CInt=2
uint8 CString=3
uint8 CLong=5

uint8 CDoubleList=10
uint8 CIntList=20
uint8 CStringList=30
uint8 CPoint2List=40
